core/ai.go

```diff
@@ -49,26 +49,10 @@ type AIModelConfig struct {
 	Warm              bool                     `json:"warm,omitempty"`
 	PricePerUnit      JSONRat                  `json:"price_per_unit,omitempty"`
 	PixelsPerUnit     JSONRat                  `json:"pixels_per_unit,omitempty"`
+	Currency          string                   `json:"currency,omitempty"`
 	OptimizationFlags worker.OptimizationFlags `json:"optimization_flags,omitempty"`
 }
 
-func (config *AIModelConfig) UnmarshalJSON(data []byte) error {
```

Comment:
@victorges I removed the default behavoir since I figured we would like to use the default specified by the command line when `PixelsPerUnit` is not supplied.

Reply:
Yeah I think that's valid. These configs could be treated as "overrides" for the specific capability right.